instructions/steps.md

```diff
@@ -1023,488 +1025,62 @@ This is a slice notation.
+It will fail because we have not completed all sections.
 
+```bash
+make duvet_report
 ```
 
-Putting that all together we get.
+You have completed section 2.5!
+Now we know that every requirement in that section is annotated
+and we know that our implementation follows it's Dafny specification.
 
-<!-- !test check MultiRegionAwsKmsIdentifier?Correct complete -->
-```dafny
+Before we move on,
+remember that we could put `duvet` into our CI build process.
+This would create a mechanism to ensure that every requirement is meet.
+It also is a way to track changes
+if new requirements are added the project.
 
-  lemma MultiRegionAwsKmsIdentifier?Correct(identifier: string)
-    requires ParseAwsKmsIdentifier(identifier).Success?
+Why don't we execute it?!
 
-    //= aws-kms-key-arn.txt#2.9
-    //= type=implication
-    //# If the input starts with "arn:", this MUST return the output of
-    //# identifying an an AWS KMS multi-Region ARN (Section 2.8) called with this
-    //# input.
-    ensures "arn:" <= identifier
-      ==>
-        var arnIdentifier := ParseAwsKmsIdentifier(identifier).value;
-        MultiRegionAwsKmsIdentifier?(arnIdentifier) == MultiRegionAwsKmsArn?(arnIdentifier.a)
-
-    //= aws-kms-key-arn.txt#2.9
-    //= type=implication
-    //# If the input starts with "alias/", this an AWS KMS alias and
-    //# not a multi-Region key id and MUST return false.
-    ensures "alias/" <= identifier
-      ==>
-        var resource := ParseAwsKmsIdentifier(identifier).value;
-        !MultiRegionAwsKmsIdentifier?(ParseAwsKmsIdentifier(identifier).value)
-
-    //= aws-kms-key-arn.txt#2.9
-    //= type=implication
-    //# If the input starts
-    //# with "mrk-", this is a multi-Region key id and MUST return true.
-    ensures "mrk-" <= identifier
-      ==>
-        var resource := ParseAwsKmsIdentifier(identifier).value;
-        MultiRegionAwsKmsIdentifier?(resource)
-
-    //= aws-kms-key-arn.txt#2.9
-    //= type=implication
-    //# If
-    //# the input does not start with any of the above, this is not a multi-
-    //# Region key id and MUST return false.
-    ensures
-        && !("arn:" <= identifier )
-        && !("alias/" <= identifier )
-        && !("mrk-" <= identifier )
-      ==>
-        var resource := ParseAwsKmsIdentifier(identifier);
-        !MultiRegionAwsKmsIdentifier?(resource.value)
-  {}
+In our makefile there are targets for each supported runtime.
+Feel free to play around with other values!
 
+```bash
+make execute
 ```
 
-## Step 21
-
-Let's run Duvet one last time:
+If you would like to investigate the source that Dafny produces
+this will output the source to a `runtimes` directory.
 
 ```bash
-make duvet_report
+make compile
 ```
 
-Success! The command no longer fails
-(if you don't believe the lack of error messages, you can check with `echo $?`)
-and the report is a happy sea of green.
-Congratulations on your formally-verified implementation!
-
-## Extra Credit
+## More Extra Credit
```

Comment:
Why is this "more" now?

Reply:
because I was going to split it... changed back